src/ui/public/timepicker/relative_panel/kbn_timepicker_relative_panel.html

```diff
@@ -62,7 +73,13 @@
             ng-change="formatRelative({key:'from'})"
           >
           <span class="kuiCheckBoxLabel__text">
-            round to the {{units[relative.from.unit.substring(0,1)]}}
+              <FormattedMessage
+                i18n-id="common.ui.timepicker.roundTo"
+                i18n-default-message="round to the {unit}"
```

Comment:
I would suggest we add a `i18n-description` here something like `{unit} will be a translated time unit, e.g. "hour", "minute"` to make the context more clear for that sentence.